python/src/aiconfig/editor/client/src/utils/api.ts

```diff
@@ -14,6 +14,7 @@ export const ROUTE_TABLE = {
   CANCEL: urlJoin(API_ENDPOINT, "/cancel"),
   CLEAR_OUTPUTS: urlJoin(API_ENDPOINT, "/clear_outputs"),
   DELETE_PROMPT: urlJoin(API_ENDPOINT, "/delete_prompt"),
+  GET_AICONFIGRC: urlJoin(API_ENDPOINT, "/get_aiconfigrc"),
```

Comment:
What does rc mean?

Reply:
fairly certain that `rc` stands for run commands. Most people will have multiple of these on their system already, you can see which rc files you have on your own computer by running this in terminal `ls -a $HOME | grep rc`.

To define if telemetry is to be disabled, user will need to define their own `.aiconfigrc`. Will need to add this to docs as well